stores/migrations.go

```diff
@@ -98,8 +134,45 @@ func migrateShards(ctx context.Context, db *gorm.DB, l glogger.Interface) error
 }
 
 func performMigrations(db *gorm.DB, logger glogger.Interface) error {
+	migrations := []*gormigrate.Migration{
+		{
+			ID: "0001_first",
+			Migrate: func(tx *gorm.DB) error {
+				return performMigration0001_first(tx, logger)
+			},
+			Rollback: nil,
+		},
+	}
+
+	// Create migrator.
+	m := gormigrate.New(db, gormigrate.DefaultOptions, migrations)
```

Comment:
Does it make sense to add some distributed locking now that we're changing it? Making sure we don't run migrations at the same time is probably a good idea.